tools/internal_ci/macos/grpc_interop.sh

```diff
@@ -18,6 +18,6 @@ set -ex
 # change to grpc repo root
 cd $(dirname $0)/../../..
 
-source tools/internal_ci/helper_scripts/prepare_build_interop_rc
+source tools/internal_ci/helper_scripts/prepare_build_mac_interop_rc
```

Comment:
consistency: for linux we include both standard and interop rc file:

```
source tools/internal_ci/helper_scripts/prepare_build_linux_rc
source tools/internal_ci/helper_scripts/prepare_build_interop_rc
```

https://github.com/grpc/grpc/blob/master/tools/internal_ci/linux/grpc_interop_tocloud.sh#L24
That allows avoiding some duplication of the install scripts and helps maintain a more standard environment.

It looks like you've copy-pasted the software installation from `tools/internal_ci/helper_scripts/prepare_build_macos_rc`